REST connector: update documentation for oauth (#1535)
camunda/camunda-platform-docs: versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/rest.md

1. Follow our [guide for creating secrets](../../console/manage-clusters/manage-secrets.md).
2. Name your secret (i.e `OAUTH_TOKEN_ENDPOINT`) so you can reference it later in the Connector.

#### Configure the OAuth Token

Select the **REST Connector** and fill out the following properties under the **Authentication** section:

1. Click **OAuth 2.0** in the **Authentication** section.
2. Set **OAuth Token Endpoint** to the secret you created (i.e. `secrets.OAUTH_TOKEN_ENDPOINT`).
3. Set **Client ID** to the secret you created (i.e. `secrets.CLIENT_ID`).
4. Set **Client secret** to the secret you created (i.e. `secrets.CLIENT_SECRET`).
5. Set **Scopes** (i.e. `read:clients`). It is an optional field. Depending on the oauth client you're using, you should fill this field or not.
6. Set **Audience** to the secret you created (i.e. `secrets.AUDIENCE`). It is an optional field. Depending on the OAuth provider you're using, you should fill this field or not.
7. Choose **Client Authentication** from the dop-down list (i.e. `Send client credentials in body`).

![rest Connector bearer token auth](../img/connectors-rest-oauth-token.png)

> You can find more information about how the oauth works by consulting the [RFC reference](https://www.rfc-editor.org/rfc/rfc6749#section-4.4).

Reviewer: ```suggestion
You can find more about the OAuth client credentials flow at the [RFC reference](https://www.rfc-editor.org/rfc/rfc6749#section-4.4).
```
